engines/supernova/game-manager.cpp

```diff
@@ -555,13 +562,24 @@ bool GameManager::isNullObject(Object *obj) {
 void GameManager::sentence(int number, bool brightness) {
 	if (number < 0)
 		return;
+
+	_ttsMan = g_system->getTextToSpeechManager();
```

Comment:
Same remark as above. Either set the `_ttsMan` in the `GameManager` constructor, or change it to a local variable.